app/controllers/application_controller/ci_processing.rb

```diff
@@ -651,6 +651,214 @@ def evacuate_form_fields
     }
   end
 
+  def associate_floating_ip_vms
+    assert_privileges("instance_associate_floating_ip")
+    recs = find_checked_items
+    recs = [params[:id].to_i] if recs.blank?
+    @record = find_by_id_filtered(VmCloud, recs.first)
+    if @record.supports_associate_floating_ip? && @record.ext_management_system.present?
+      if @explorer
+        associate_floating_ip
+        @refresh_partial = "vm_common/associate_floating_ip"
+      else
+        render :update do |page|
+          page << javascript_prologue
+          page.redirect_to :controller => 'vm',
+                           :action     => 'associate_floating_ip',
+                           :rec_id     => @record.id,
+                           :escape     => false
+        end
+      end
+    else
+      add_flash(_("Unable to associate %{floating_ip} with %{instance} \"%{name}\": %{details}") % {
```

Comment:
We want to avoid using `ui_lookup()` in cases where the argument is just a static string. In this case, the string
can be just:

```ruby
_("Unable to associate Floating IP with Instance \"%{name}\": %{details}")
```

Similarly for other occurences of `ui_lookup()` in this PR.